test_runner/regress/test_pg_regress.py

```diff
@@ -104,7 +104,12 @@ def post_checks(env: NeonEnv, test_output_dir: Path, db_name: str, endpoint: End
         if env.pg_version <= PgVersion.V15 and r[0] == "i":
             unlogged_relation_files.append(f"{r[1]}_init")
 
-    ignored_files = unlogged_relation_files
+    ignored_files = unlogged_relation_files + [
```

Comment:
@hlinnaka could you please think about whether it's expected that we see these files differ (by being absent) when doing a `check_restored_datadir_content` after `test_isolation`?  These differences don't exist after a normal run of pg_regress, but I presume that's because they're just not being "touched" in test_isolation?

(I only have a vague grasp of what's going on inside these postgres suites)